app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -4,20 +4,41 @@ import { PageSection } from 'components/section';
 import { Sentiment } from 'components/sentiment';
 import { ContentListActionBar } from 'components/tool-bar';
 import { useElastic } from 'features/my-searches/hooks';
-import { determinePreview } from 'features/utils';
 import React from 'react';
 import { FaPlay, FaStop } from 'react-icons/fa';
 import { FaBookmark } from 'react-icons/fa6';
 import { useNavigate, useParams } from 'react-router-dom';
 import { toast } from 'react-toastify';
 import { useContent, useFilters, useLookup } from 'store/hooks';
 import { useProfileStore } from 'store/slices';
-import { Checkbox, Col, IContentModel, Loading, Row, Show } from 'tno-core';
+import {
+  Checkbox,
+  Col,
+  ContentTypeName,
+  IContentModel,
+  Loading,
+  Row,
+  Settings,
+  Show,
+} from 'tno-core';
 
 import { AdvancedSearch } from './components';
 import { Player } from './player/Player';
 import * as styled from './styled';
-import { filterFormat, formatSearch } from './utils';
+import { filterFormat } from './utils';
+
+const formatDate = (date: string) => {
```

Comment:
Looks like a util function.  If it doesn't need to be part of the component, extract it.